.travis.yml

```diff
@@ -105,87 +100,87 @@ jobs:
-  #     - secure: "WzNy/QJoUukfd9G6f62VasEV1aZneJOe0r5tlIiHzKjyFi6tNlizBiIsoOosbUoW8qaEf8xY2+H1pLeroBWcswi1hMzYNPmYHu8HfWhD64/yMDkTz6obpzWtvge88ZsjCHzXcgIu8/8AhOgOZpyXwmgS+GMTfvL4quo/k4c3ZsSwv/HHnH3Uh2HUnMK+AFY16ZIGnKAbP7vf8S5lPgyB3w+FjzyAPxYxpc74V7yOg5uyJW8W9mews/wyAKnWMyEWBUBpieOXvCT52R1jQNplAootSSrdm5Wj3hQRhsH5b/hJUeHX9ex586sn4zXbb7AOfVuK0y8TVovdT9Ld2osKATfXYS4I3yVX6ZdwAZ0wYd4znw+7fVxi+qbZ+46cKzDEOZ+0jN+bzn9SFlOy4Z0VUIZEM9ooQPpQJ9XXlglmHZdnwxD1LUITP3H4vljPdcl5+oz10E7zWN/epMClqJXbRds1RDZ+l9m1lEpt05DCYPWW61+Dca10szaxRavY390wkz9JqD2f92aqcTmYhVu2X46zL3pZONLpd8DlQv2Je/3ik7E28V875SDy/4mMSZyi+RhrG6Ob5A0zTXeiZArVKkGinlfyhcAIgu7t0brd0l7hL0eNqOpqU6Jnqz5TWLwMJjqsoMuSiheYl1avLMQYTg1EDA56dMloZEh0jdihYfM="
-  #     - secure: "pmnIVxqixHvgidZRXXmWKUfuyAeBun1eOzhP79onCFTazGRnMG+Lv+c4422nHUDA9KsmYPezjk3ESfd8EGlfL0ZFQbL45eQp3oGcbN9to6wDj8g6xyrr+tG9GJ4jirO5DlLZ5I0y3WywmE+anzbFdjV0+sWTe1QWxtzqbySCwewPThBvn/iOdJMaPh7S328W7WxkD095ZCC2pQx42Mwfforw+1Jg6yd2ZI3W4b0+KsrTBveeyfj0A7F3e7uTrA+zXst26sYY37zlT/IYT5hgpQiGqbWpnoTrYLvgjdnLxSz43zWKY/YDxFU7DY1t0HIKC4N9vWynjcx1R8VjWT4SCy5f1TPHNvNt5zJUBJ59zUBuTL8Yi6FfMqDqHRLq6W8/bMJe0QSrjJFocWIs+YD+AlCbTS6rhpynzsrbS/GHo6KjVkFhXIRVjp798IFLHsJQ2JonO8McWdaldQszu8O/rB1v8B9tXsUoc5e0pJFSXI9qVwPYAAMOx1bmDPOBQoapx5ciwUh0U2rPZyGDQVh+iDcGlGX1ttWlF83vaRYbzBYqzLzcckNSnHlva6F/ZwcTQsbrZSM5Trz3OUk7fWJMRuf7XjKFAEWzASFLYpdZ1EVP+vUmxpT8OzA9kUn3XMFd9sTm9v5GZ0zQscM+/WSmJsPPGwlQZRX5CtHbq7mySJ0="
-  #   install:
-  #     # Install GCP SDK if not already cached
-  #     - if [ ! -d "$HOME/google-cloud-sdk/bin" ]; then rm -rf "$HOME/google-cloud-sdk"; curl https://sdk.cloud.google.com | bash > /dev/null; fi
-  #     - source $HOME/google-cloud-sdk/path.bash.inc
-  #     - gcloud version
-  #     - pip install -r tests/requirements.txt
-  #     - ./tests/e2e/helm-preinstall.sh
-  #   before_script:
-  #     - openssl aes-256-cbc -K $encrypted_1019fd6574a7_key -iv $encrypted_1019fd6574a7_iv -in tests/e2e/e2e-gcp-sa-key.json.enc -out tests/e2e/e2e-gcp-sa-key.json -d
-  #     - gcloud auth activate-service-account --key-file tests/e2e/e2e-gcp-sa-key.json
-  #     - gcloud auth configure-docker
-  #     - ./build/docker-deploy.sh
-  #     - ./tests/e2e/before-deployment-test-helm-cleanup.sh
-  #     - ./tests/e2e/before-deployment-test-extensions-cleanup.sh
-  #   script:
-  #     - ./tests/e2e/execute-tests.sh
-  #   after_script:
-  #     - ./tests/e2e/deployment-test-cleanup.sh
+  - stage: e2e-helm-deployment-test
+    name: E2E GCP helm deployment test
+    sudo: required
+    language: python
+    python:
+      - "3.8"
+    if: (branch = master OR type = pull_request) AND fork = false
```

Comment:
What is the deployment strategy here? Test the deployment on each push to `master` or on each new release?

If you plan to run deployment tests on each push to `master` or when a PR is created, then the PR title ("deploy after each new release") is somewhat misleading.

Reply:
this is `e2e-helm-deployment-test` section

Reply:
As it turns out, this comment was pointing to the pre-PR version of the code. Either my mistake or GH is playing hide-and-seek.